Application/EdFi.Ods.Tests/EdFi.Ods.Common/Models/Domain/AssociationViewTests.cs

```diff
@@ -1512,50 +1512,50 @@ protected override void Arrange()
                         {
                             new EP(
                                 "DescriptorId",
-                                new PT(DbType.Int32, 0, 10, 0, false),
+                                new PT(DbType.Int32, 0, 0, 10, 0, false),
```

Comment:
As I mentioned in an earlier comment, move the `minLength` argument to the end of the constructor signature and revert all changes such as this.

Reply:
I moved to last as minLength parameter  and updated it for all places at the end .